token-swap/program/src/processor.rs

```diff
@@ -550,7 +553,8 @@ impl Processor {
                 to_u128(token_a.amount)?,
             )
             .ok_or(SwapError::ZeroTradingTokens)?;
-        if a_amount < to_u128(minimum_token_a_amount)? {
+        let a_amount = to_u64(a_amount)?;
```

Comment:
Was this a bug fix as a result of fuzzing?

Reply:
It actually wasn't, just an inconsistency in the code from when we transitioned to `u128`